cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -385,6 +388,17 @@ You cannot use the following reserved CIDR blocks:
 169.254.169.252/30
 ----
 
+You can configure the following private space internal CIDRs as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```

Comment:
let's document the list as "MuleSoft infrastructure CIDRs pool".

Reply:
```suggestion
You can configure the following CIDRS from the MuleSoft infrastructure CIDRs pool as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
```